reactive/k8s.py

```diff
@@ -294,19 +375,27 @@ def render_files(reldata=None):
     # definition for kubelet and proxy.
     render('docker-compose.yml', target, context)
 
-    # Source: https://github.com/kubernetes/...master/cluster/images/hyperkube
-    target = os.path.join(rendered_manifest_dir, 'master.json')
-    # Render the files/manifests/master.json that contains parameters for the
-    # apiserver, controller, and controller-manager
-    render('master.json', target, context)
-    # Source: ...master/cluster/addons/dns/skydns-svc.yaml.in
-    target = os.path.join(rendered_manifest_dir, 'skydns-svc.yml')
-    # Render files/kubernetes/skydns-svc.yaml for SkyDNS service.
-    render('skydns-svc.yml', target, context)
-    # Source: ...master/cluster/addons/dns/skydns-rc.yaml.in
-    target = os.path.join(rendered_manifest_dir, 'skydns-rc.yml')
-    # Render files/kubernetes/skydns-rc.yaml for SkyDNS pod.
-    render('skydns-rc.yml', target, context)
+    if is_leader():
+        # Source: https://github.com/kubernetes/...master/cluster/images/hyperkube  # noqa
+        target = os.path.join(rendered_manifest_dir, 'master.json')
+        # Render the files/manifests/master.json that contains parameters for
+        # the apiserver, controller, and controller-manager
+        render('master.json', target, context)
+        # Source: ...master/cluster/addons/dns/skydns-svc.yaml.in
+        target = os.path.join(rendered_manifest_dir, 'skydns-svc.yml')
+        # Render files/kubernetes/skydns-svc.yaml for SkyDNS service.
+        render('skydns-svc.yml', target, context)
+        # Source: ...master/cluster/addons/dns/skydns-rc.yaml.in
+        target = os.path.join(rendered_manifest_dir, 'skydns-rc.yml')
+        # Render files/kubernetes/skydns-rc.yaml for SkyDNS pod.
+        render('skydns-rc.yml', target, context)
+
+
+def status_set(level, message):
```

Comment:
Ahhhh i see what you did there!  And I liked it (Even though you gave me a scare)